### ISSUE TITLE
[a11y]: nsw-order utility class changes visual order without changing DOM order, leading to possible WCAG issues

### ISSUE THREAD

**Issue by JeffersonBledsoe** [digitalnsw/nsw-design-system#654]
**Describe the bug**

The [nsw-order](https://github.com/digitalnsw/nsw-design-system/blob/5e7e3107808ce82bb3a03f58d76d08465c570372/src/global/scss/helpers/_utilities.scss#L295-L309) utility class allows setting the [CSS order property](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/order) on content and changing the way the order of the pages visual appearance. An example of this class being used in such a way can be found on the [Hero Banner Landing Page Template](https://designsystem.nsw.gov.au/templates/landing/hero-banner.html) in the "NSW Connectivity Strategy" part of the page.
The issue with this is that the DOM order does not change. Many assistive technologies such as screen readers rely on the DOM or accessibility tree, causing a disconnect between the order of content visually on the page and the order to these tools. This could fail [WCAG SC 1.3.2 - Meaningful Sequence](https://www.w3.org/TR/WCAG21/#meaningful-sequence) and [SC 2.4.3 Focus Order](https://www.w3.org/TR/WCAG21/#focus-order). Some examples of exploring this topic are https://adrianroselli.com/2015/10/html-source-order-vs-css-display-order.html and https://tink.uk/flexbox-the-keyboard-navigation-disconnect.

Could we get some clarification as to whether or not this is an issue? And if it is, make a note of it in the documentation?

**To Reproduce**
Steps to reproduce the behaviour:
1. Visit https://designsystem.nsw.gov.au/templates/landing/hero-banner.html on a mobile device or in a browser window wider than 960px
2. Start a screen reader 
3. Use the screen reader to navigate through the "NSW Government Data Strategy" and "NSW Connectivity Strategy" part of the page

**Expected behaviour**

The "NSW Connectivity Strategy" grid is read from left to right.

**Actual behaviour**

The image is read aloud first before the text content, even though it visually appears next to/ after the text.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser chrome/ safari/ firefox
 - Version 143/ 26.1/ 146

**Comment by oisa** [comment]
Hey @JeffersonBledsoe - thanks so much for identifying this and passing it through to us! 

We’ve got it on our radar and will be in touch shortly with a response on `nsw-order` use. Appreciate you flagging it.
